src/main/java/com/metamx/emitter/core/ComposingEmitter.java

```diff
@@ -0,0 +1,79 @@
+/*
+* Licensed to Metamarkets Group Inc. (Metamarkets) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. Metamarkets licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package com.metamx.emitter.core;
+
+import com.metamx.common.lifecycle.LifecycleStart;
+import com.metamx.common.lifecycle.LifecycleStop;
+import com.metamx.common.logger.Logger;
+
+import java.io.IOException;
+import java.util.List;
+
+public class ComposingEmitter implements Emitter
+{
+  private static Logger log = new Logger(ComposingEmitter.class);
+
+  private final List<Emitter> emitters;
+
+  public ComposingEmitter(List<Emitter> emitters)
```

Comment:
option 1) Remove ComposingEmitterConfig , then https://github.com/druid-io/druid/pull/1699/files#diff-aad34d56314f41ec5cda04efa6e3b318R64 will take Properties object , fetch config property value string, split it into list and we lose the goodness that jackson automatically converted the string to list.

option 2) I couldn't really understand how to make it work, can you pls show me what the code in https://github.com/druid-io/druid/pull/1699/files#diff-aad34d56314f41ec5cda04efa6e3b318R64 will look like if you had List&lt;Emitter&gt; inside ComposingEmitterConfig  and not List&lt;String&gt; and what additional stuff will be inside ComposingEmitterConfig   ?